package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "phantomjs",
-  "version": "1.9.19",
+  "version": "2.1.0",
```

Comment:
ya....i'm on the fence about whether it should be `2.1.0` vs `2.1.1`...technically this is more correct, but we're going to get a bunch of issues filed for "update to 2.1.1"